apps/src/code-studio/components/progress/progress_dot.jsx

```diff
@@ -185,7 +187,8 @@ export const ProgressDot = React.createClass({
     const showLevelName = /(named_level|peer_review)/.test(level.kind) && this.props.courseOverviewPage;
     const isPeerReview = level.kind === 'peer_review';
     const iconForLevelStatus = !isUnplugged && this.props.courseOverviewPage && this.getIconForLevelStatus(level);
-    const levelUrl = level.locked ? undefined : level.url + location.search;
+    const isLocked = level.locked || level.status === LevelStatus.locked;
```

Comment:
To a completely uninformed reader (me) why do we have both `level.locked` and `level.status === LevelStatus.locked`?